src/main/scala/scalaz/parsers/transform.scala

```diff
@@ -68,6 +82,16 @@ object Transform {
       override def combine[A, B, C](ab: A => F[B], ac: A => F[C]): A => F[B /\ C] =
         a => F.ap(ab(a))(F.map(ac(a))(c => b => (b, c)))
 
+      override def cross[A, B, C, D](
+        ab: A => F[B],
+        cd: C => F[D]
+      ): A /\ C => F[(C /\ B) /\ (A /\ D)] =
```

Comment:
It's all great until this point, where we expose this method in public API.
But what is the meaning of it?
Do you know of any example outside of refactoring `conjunction` internals where it can be used?
Thanks!